src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEParameterSymbol.cs

```diff
@@ -645,6 +647,44 @@ internal override bool IsCallerMemberName
             }
         }
 
+        internal override AttributeAnnotations FlowAnalysisAnnotations
+        {
+            get
+            {
+                const WellKnownAttributeFlags notNullWhenFalse = WellKnownAttributeFlags.NotNullWhenFalse;
+                const WellKnownAttributeFlags ensuresNotNull = WellKnownAttributeFlags.EnsuresNotNull;
+
+                // PROTOTYPE(NullableReferenceTypes): the flags could be packed more
+                if (!_packedFlags.TryGetWellKnownAttribute(notNullWhenFalse, out bool hasNotNullWhenFalse) ||
+                    !_packedFlags.TryGetWellKnownAttribute(ensuresNotNull, out bool hasEnsuresNotNull))
```

Comment:
To avoid race conditions, we probably need to get the two bits atomically (and then set the two bits atomically), or get and set the two bits independently.